src/NetworkFunction/custom/Get-AzNetworkFunctionAzureTrafficCollector.ps1

```diff
@@ -0,0 +1,200 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You don't need to custom this file if you just want to add `src/NetworkFunction/custom/GetAzNetworkFunctionAzureTrafficCollector_List.cs`. Our generated code will generate this with your custom parameter set cmdlet.

Reply:
Removed this file.